editor/src/components/editor/actions/action-utils.ts

```diff
@@ -131,7 +132,6 @@ export function isTransientAction(action: EditorAction): boolean {
     case 'UNWRAP_LAYOUTABLE':
     case 'UPDATE_JSX_ELEMENT_NAME':
     case 'SET_ASPECT_RATIO_LOCK':
-    case 'SAVE_CURSOR_POSITION':
     case 'SET_CODE_EDITOR_THEME':
```

Comment:
while we are here... this also sounds like it shouldn't be in the undo flow? :)

Reply:
Yeah... this is part of project "Fix the damn undo flow". The problem is "Transient" functions are used to label things that we want to add to the undo stack and that we want to trigger a save, both of which should definitely be separate things. I'm going to say we leave this until we finally get a chance to tackle that as a whole.